pkg/artifactory/resource/repository/remote/resource_artifactory_remote_repository_test.go

```diff
@@ -875,6 +876,56 @@ func TestAccRemoteRepository_generic_with_propagate(t *testing.T) {
 	})
 }
 
+func TestAccRemoteRepository_gems_with_propagate_fails(t *testing.T) {
```

Comment:
This should be applied to all `RepoTypesLikeBasic` repo types.

Reply:
Agree.